doc/changelogs/CHANGELOG_V24.md

```diff
@@ -0,0 +1,497 @@
+# Node.js 24 ChangeLog
+
+<!--lint disable maximum-line-length no-literal-urls prohibited-strings-->
+
+<table>
+<tr>
+<th>Current</th>
+</tr>
+<tr>
+<td>
+<a href="#24.0.0">24.0.0</a><br/>
+</td>
+</tr>
+</table>
+
+* Other Versions
+  * [23.x](CHANGELOG_V23.md)
+  * [22.x](CHANGELOG_V22.md)
+  * [21.x](CHANGELOG_V21.md)
+  * [20.x](CHANGELOG_V20.md)
+  * [19.x](CHANGELOG_V19.md)
+  * [18.x](CHANGELOG_V18.md)
+  * [17.x](CHANGELOG_V17.md)
+  * [16.x](CHANGELOG_V16.md)
+  * [15.x](CHANGELOG_V15.md)
+  * [14.x](CHANGELOG_V14.md)
+  * [13.x](CHANGELOG_V13.md)
+  * [12.x](CHANGELOG_V12.md)
+  * [11.x](CHANGELOG_V11.md)
+  * [10.x](CHANGELOG_V10.md)
+  * [9.x](CHANGELOG_V9.md)
+  * [8.x](CHANGELOG_V8.md)
+  * [7.x](CHANGELOG_V7.md)
+  * [6.x](CHANGELOG_V6.md)
+  * [5.x](CHANGELOG_V5.md)
+  * [4.x](CHANGELOG_V4.md)
+  * [0.12.x](CHANGELOG_V012.md)
+  * [0.10.x](CHANGELOG_V010.md)
+  * [io.js](CHANGELOG_IOJS.md)
+  * [Archive](CHANGELOG_ARCHIVE.md)
+
+<a id="24.0.0"></a>
+
+## 2025-05-06, Version 24.0.0 (Current), @RafaelGSS and @juanarbol
+
+We’re excited to announce the release of Node.js 24! This release brings
+several significant updates, including the upgrade of the V8 JavaScript
+engine to version 13.6 and npm to version 11.0.0. Starting with
+Node.js 24, support for MSVC has been removed, and ClangCL is now required
+to compile Node.js on Windows. The `AsyncLocalStorage` API now uses
+`AsyncContextFrame` by default, and `URLPattern` is available globally.
+These changes, along with many other improvements, continue to push the
+platform forward.
+
+As a reminder, Node.js 24 will enter long-term support (LTS) in October,
+but until then, it will be the "Current" release for the next six months.
+We encourage you to explore the new features and benefits offered by this
+latest release and evaluate their potential impact on your applications.
+
+## Notable Changes
+
+### V8 13.6
+
+The V8 engine is updated to version 13.4, which includes several new
+JavaScript features:
+
+* [RegExp.escape](https://chromestatus.com/feature/5179154657705984) - A static method that takes a string and returns an escaped version that may be used as a pattern inside a regular expression.
+* [Float16Array](https://chromestatus.com/feature/5164400693215232) - Adds the Float16Array typed array.
```

Comment:
Do we want to keep this in the changelog? AFAICT it's not exposed on the global object
```suggestion
```

Reply:
I added a comment in the notable change mentioning it